src/doc/en/developer/walkthrough.rst

```diff
@@ -62,35 +48,35 @@ So first, open a shell (for instance, Terminal on Mac) and check that
     concept guides. See 'git help <command>' or 'git help <concept>'
     to read about a specific subcommand or concept.
 
-
-Don't worry about the giant list of subcommands. You really only need
-a handful for effective development, and we will walk you through them
-in this guide. If you got a "command not found" error, then you don't
-have git installed. Now is the time to install it; see
+Don't worry about the giant list of subcommands. You really only need a handful
+of them for effective development, and we will walk you through them in this
+guide. If you got a "command not found" error, then you don't have Git
+installed; now is the time to install it. See
 :ref:`chapter-git-setup` for instructions.
 
-Because we also track who does changes in Sage with git, you must tell
-git how you want to be known. This only needs to be done once::
+Because we also track who does what changes with Git, you must tell
+Git how you want to be known. Check if Git knows you::
 
-    [user@localhost]$ git config --global user.name "Your Name"
-    [user@localhost]$ git config --global user.email you@yourdomain.example.com
-
-If you have multiple accounts / computers use the same name on each of
-them. This name/email combination ends up in commits, so do it now
-before you forget!
+    [alice@localhost ~]$ git config --global user.name
```

Comment:
I may replace "localhost" with "desktop" or "laptop" as these appear elsewhere... Do you want?

Reply:
`[alice@localhost directory]` is very recognizable as a typical UNIX prompt.

And I like the use of `alice` vs `bob` for distinguishing between the author/reviewer roles

Reply:
> The "user@localhost" or whatever is not adding anything for me, and only confused me (when I first looked at the docs 2 years ago my question was "I'm supposed to run these commands as user and not root?!").

Then "alice@localhost" is a slight improvement for you. 

> But that may be because the windows terminal doesn't show this info, so I'm not used to it - while everyone else might be. 

Yes. I think linux is the norm at least for Sage development. It seems you are special in a way because windows users are not used to the notion of root versus normal user.

> Are there not established conventions how this is done in linux docs?

I don't know. We may use simple prompt "$". But as I said above, I didn't introduce it, and I don't feel like making that change here. So please make your own PR for that after this PR.